### ISSUE TITLE
Is --num_alignments 1 the default parameters?

### ISSUE THREAD

**Issue by 473021677** [sortmerna/sortmerna#432]
Hi,
I am using SortMeRNA v4.3.6 to filter the metatranscriptomic reads. I have one doubt about the parameters. I am not sure whether "--num_alignments 1" is the default parameter. I need your help. Thanks.

Best regards,
Yang Yuan

**Comment by biocodz** [comment]
Yes, 1 is the default. Note that '-num_alignments' only works when used with '-no-best' option. Otherwise it is ignored, and the program searches the best alignments using an exhaustive search

**Comment by 473021677** [comment]
Thanks for your explanation. I have another question about the "--out2" parameter. I am not sure whether it only work if a single reads file is provided. It it is true, this parameter may be ignored when the paired-end reads files are provided. I need your help. Thanks.

Best regards,
Yang Yuan

**Comment by biocodz** [comment]
'-out2' can be used with either single file or two files. Its main purpose is to instruct the program to always keep reads of different sense separately. The file [test.jinja](https://github.com/sortmerna/sortmerna/blob/master/scripts/test.jinja) has many usage examples, and lists expected output e.g. [test 16_6](https://github.com/sortmerna/sortmerna/blob/master/scripts/test.jinja#L1011) generates [files listed here](https://github.com/sortmerna/sortmerna/blob/master/scripts/test.jinja#L1043)

**Comment by 473021677** [comment]
Hi,
I have run SortMeRNA v4.3.6 with and without the "--out2" parameter. And I have found that one rRNA-filtered fastq file (JZ2_202101_fastp_rm_rRNA_v2.fq) has been generated for the run without "--out2" parameter, but two rRNA-filtered fastq files (JZ2_202101_fastp_rm_rRNA_v3_fwd.fq, JZ2_202101_fastp_rm_rRNA_v3_rev.fq) have been generated for the run with "--out2" parameter. I have compared the file sizes between the "JZ2_202101_fastp_rm_rRNA_v2.fq" file and the sum of the "(JZ2_202101_fastp_rm_rRNA_v3_fwd.fq" and "(JZ2_202101_fastp_rm_rRNA_v3_rev.fq" files, and found that there are slight differences between them. I am not sure whether running SortMeRNA v4.3.6 with or without the "--out2" parameter impact the metatranscriptomic mapping results. I need your help. Thanks.

Best regards,
Yang Yuan

**Comment by biocodz** [comment]
Could you publish the [execution trace](https://github.com/sortmerna/sortmerna/?tab=readme-ov-file#execution-trace)?
